services/gateway/apis/http-version3/swagger/definitions/blockchainApps.json

```
@@ -242,11 +238,6 @@
 					}
 				}
 			},
```

This was correct before.
```suggestion
			},
			"appPage": {
				"type": "string",
				"example": "https://lisk.com",
				"description": "URL to the blockchain application's frontend, if exists"
			},
```